config/manager/manager.template.yaml

```diff
@@ -30,6 +30,7 @@ spec:
         env:
           - name: KVM_INFO_IMAGE
           - name: VALIDATOR_IMAGE
+            value: $VALIDATOR_IMG
```

Comment:
I think the variable needs to be quoted `"$VALIDATOR_IMG"`. Otherwise, if it was empty, the yaml would not be valid.

Reply:
Good catch, changed now